Orange/widgets/visualize/owdistributions.py

```diff
@@ -91,9 +92,9 @@ class Inputs:
     settingsHandler = settings.DomainContextHandler(
         match_values=settings.DomainContextHandler.MATCH_VALUES_ALL)
     #: Selected variable index
-    variable_idx = settings.ContextSetting(-1)
+    var = settings.ContextSetting(None)
     #: Selected group variable
-    groupvar_idx = settings.ContextSetting(0)
+    cvar = settings.ContextSetting(None)
```

Comment:
While you're refactoring, could you also rename `cvar` to `group_var`?

Reply:
Fixed.